tests/single-node/init.sh

```diff
@@ -0,0 +1,19 @@
+die() {
+    exit 1
+}
+
+echo "Creating loopback block-device and configuring VM"
+sudo truncate -s 256G /kube-lvm || die
```

Comment:
Would it make sense to require this whole script as root, instead of using `sudo` everywhere?

Also, likely should be part of `eve/...` (cfr. inventory comment earlier)?